Dockerfile

```diff
@@ -1,3 +1,18 @@
+FROM node:6 as frontend
```

Comment:
So, does this beast come with a decent and recent version of yarn? 

One thing that strikes me as a novice is that it's there's no comments. 
The appropriate comment could be,
```
First, build the frontend code static assets based on Node 6. The culmination 
is the execution of `bin/build_frontend.sh`. 
```

Reply:
as of the current: https://github.com/nodejs/docker-node/blob/7701eea6fe125530894b3e83f1e9d385e9ee509f/6.11/Dockerfile#L41